src/main/webapp/css/main.css

```diff
@@ -142,6 +142,22 @@ header li a {
   height: 100px;
   width: 100px;
   top:78%;
-  padding-left:43%;
+  padding-left:41%;
   position:fixed;
-}
\ No newline at end of file
+  opacity: 0;
+  display: inline-block;
+  animation: animateTix 5s linear infinite;
```

Comment:
Okay I'll probably change this infinite (ticket keeps blinking/fading in continuously) to forward (ticket fades in once) because it looks slightly annoying